client/dashboard/app/dataviews-overrides.scss

```diff
@@ -27,3 +27,12 @@ a.dataviews-url-field {
 		text-decoration: none;
 	}
 }
+
+.dataviews-view-table tbody td {
+	vertical-align: middle;
```

Comment:
I just remembered this long-forgotten issue DOTDASH-84; should we do it in Core as proposed 🙈 @Automattic/lego